tests/unit/core/test_application.py

```diff
@@ -40,4 +43,4 @@ def test_post_process_urls_adds_decorator(self):
         processed_patterns = self.myapp.post_process_urls([pattern])
 
         self.myapp.get_url_decorator.assert_called_once_with(pattern)
-        self.assertEqual(processed_patterns[0].callback, 'fake_callback')
+        self.assertEqual(processed_patterns[0].callback, fake_callback)
```

Comment:
Good finding. Let's extract this change into the separate PR.

Reply:
Done in https://github.com/django-oscar/django-oscar/pull/3556